aviary/utils/conflict_checks.py

```diff
@@ -3,13 +3,10 @@
 from aviary.variable_info.variables import Aircraft
 
 
-def check_fold_location_definition(inputs, options: AviaryValues):
+def check_fold_location_definition(inputs, choose_fold_location, has_strut):
```

Comment:
these might need defaults, these conflict checks were originally intended to have a standardized interface.

Reply:
What default values would you recommend for the 2 options?

Reply:
originally, all of the conflict checks (half dozen or so) all took `inputs` and `options` as positional args. Since this is the only one left, maybe it would be fine to just remove `inputs` and have it just take the variables it needs

Reply:
Done.